envi/__init__.py

```diff
@@ -44,39 +44,74 @@
 
 ARCH_MASK        = 0xffff0000   # Masked into IF_FOO and BR_FOO values
 
+# Code Maturity Ratings.  Also used for Unittest categories
+MAT_NONE = 0            # does not exist
+MAT_ALPHA = 1           # code exists, but immature.  no promises
+MAT_BETA = 2            # kinda works, not overly confident
+MAT_TESTING = 3         # works ok, initial RC
+MAT_PROD = 4            # production
+
+maturity = {v:k for k,v in globals().items() if k.startswith('MAT_')}
```

Comment:
I think now that we've settled on some boolean flags and semver, we can remove these enums?

Reply:
good call.